.dockerignore

```diff
@@ -0,0 +1 @@
+/bin
```

Comment:
this is proabably not required.

Reply:
Any reason not to have it? `hack/build.sh` puts stuff there, which inflates the context of docker builds.